atlassian/__init__.py

```diff
@@ -7,5 +7,16 @@
 from .crowd import Crowd
 from .service_desk import ServiceDesk
 from .marketplace import MarketPlace
+from .jira8 import Jira8
 
-__all__ = ['Confluence', 'Jira', 'Bitbucket', 'Portfolio', 'Bamboo', 'Stash', 'Crowd', 'ServiceDesk', 'MarketPlace']
+__all__ = [
+    'Confluence',
+    'Jira',
```

Comment:
Hi! 
Looks like need to add Jira8 module

Reply:
Fixed. I forgot. ))